draft-ietf-quic-transport.md

```diff
@@ -2883,16 +2884,21 @@ needing acknowledgement are received.  The sender can use the receiver's
 Strategies and implications of the frequency of generating acknowledgments are
 discussed in more detail in {{QUIC-RECOVERY}}.
 
+ACK-only packets are only sent in response to ACK-eliciting packets, so a
+receiver that is only sending ACK frames will only receive acknowledgements
+for its packets if the sender includes them in packets with non-ACK frames.
```

Comment:
```suggestion
for its packets if the sender includes them in packets with ACK-eliciting frames.
```
cover PADDING-only (as pointless as it might seem).

Reply:
This section is the only use of ACK-eliciting, so I define it at the beginning of the section now.